tools/test_set_firewall_ns.py

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env python3
+
+import sys
+import json
+from cloudcix_primitives import set_firewall_ns
+
```

Comment:
Please add a prerequisites comment here. If you like, you can use this one:

https://github.com/CloudCIX/primitives/blob/main/tools/test_nat_firewall_ns.py#L9 (has all the prerequisites we need).

Reply:
Oh I misunderstood, I thought this was a reference to the comments around the imports (stdlib, local etc). My bad!